rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistSupport.java

```diff
@@ -0,0 +1,24 @@
+package co.rsk.peg.whitelist;
+
+import java.math.BigInteger;
+import org.ethereum.core.Transaction;
+
+/**
+ * Interface for the whitelist support. This interface is used to add, remove and get whitelist entries.
+ */
+public interface WhitelistSupport {
+
+    Integer getLockWhitelistSize();
+
+    LockWhitelistEntry getLockWhitelistEntryByIndex(int index);
+
+    LockWhitelistEntry getLockWhitelistEntryByAddress(String addressBase58);
```

Comment:
Will these get methods return a `null` reference in the case of not being able to retrieve the `LockWhitelistEntry`? What about returning an `Optional<LockWhitelistEntry>` and if the get fails return an `Optional.empty()`?

Reply:
That's a good point, I'll create a separate task to do that change